etl/steps/data/garden/irena/2023-06-26/renewable_electricity_capacity.meta.yml

```diff
@@ -0,0 +1,117 @@
+tables:
+  renewable_electricity_capacity:
+    variables:
+      bagasse:
+        title: Bagasse capacity
```

Comment:
maybe worth adding descriptions for these? 

> Bagasse is a byproduct from the process of sugar cane refining. After the juice has been extracted from the sugar cane, what's left over is a fibrous material called bagasse. It is often used as a biofuel for the production of heat, energy, and electricity, and in the manufacture of pulp and building materials.

Reply:
Thanks, good idea. Where did you get this definition from?

Reply:
GPT-4 :) you'd of course need to check it's accurate but it's usually pretty good at definitions. The code looks good otherwise, better than anything I ever write :)